azkaban-web-server/src/main/java/azkaban/webapp/WebMetrics.java

```diff
@@ -42,13 +44,24 @@
 
   WebMetrics() {
     registry = MetricsManager.INSTANCE.getRegistry();
-    setupAllMetrics();
+    setupStaticMetrics();
   }
 
-  private void setupAllMetrics() {
+  private void setupStaticMetrics() {
     webGetCall = MetricsUtility.addMeter("Web-Get-Call-Meter", registry);
     webPostCall = MetricsUtility.addMeter("Web-Post-Call-Meter", registry);
-    MetricsUtility.addLongGauge("fetchLogLatency", logFetchLatency, registry);
+    MetricsUtility.addGauge("fetchLogLatency", registry, logFetchLatency::get);
+  }
+
+  public void addExecutorManagerMetrics(ExecutorManager executorManager) {
+    MetricsUtility.addGauge("WEB-NumQueuedFlows", registry, executorManager::getQueuedFlowSize);
+
+    /**
+     * TODO: Currently {@link ExecutorManager#getRunningFlows()} includes both running and queueing jobs.
+     * We need to differentiate it in future.
```

Comment:
Yes. I intended to say here that this function, getRunningFlows, should be modified or refactored, in order to get really running flows. Comments revisioned.